sceneview/src/main/java/io/github/sceneview/node/Node.kt

```diff
@@ -688,24 +688,69 @@ open class Node(
         }
     }
 
-    // TODO
-//
-//    /**
-//     * ### Rotates the node to face a point in world space
-//     *
-//     * World-space up (0, 1, 0) will be used to determine the orientation of the node around the
-//     * direction.
-//     *
-//     * @param position The position to look at in world space
-//     * @param up The up direction will determine the orientation of the node around the direction
-//     */
-//    fun lookAt(position: Position, up: Direction = Direction(y=1.0f), smooth: Boolean = false) {
-//        if(smooth) {
-//            smooth(quaternion = rotation(lookAt(this.worldPosition, position, up)).toQuaternion())
-//        } else {
-//            transform(rotation = rotation(lookAt(this.worldPosition, position, up)).toQuaternion())
-//        }
-//    }
+    /**
+     * ### Rotates the node to face a point in world-space
+     *
+     * @param targetPosition The target position to look at in world space
+     * @param upDirection The up direction will determine the orientation of the node around the direction
+     * @param smooth Whether the rotation should happen smoothly
+     */
+    fun lookAt(
+        targetPosition: Position,
+        upDirection: Direction = Direction(y = 1.0f),
+        smooth: Boolean = false
+    ) {
+        val newQuaternion = lookAt(
+            worldPosition,
+            targetPosition,
+            upDirection
+        ).toQuaternion()
+        if (smooth) {
+            smooth(quaternion = newQuaternion)
+        } else {
+            transform(quaternion = newQuaternion)
+        }
+    }
+
+    /**
+     * ### Rotates the node to face another node
+     *
+     * @param targetNode The target node to look at
+     * @param upDirection The up direction will determine the orientation of the node around the direction
+     * @param smooth Whether the rotation should happen smoothly
+     */
+    fun lookAt(
+        targetNode: Node,
+        upDirection: Direction = Direction(y = 1.0f),
+        smooth: Boolean = false
+    ) = lookAt(targetNode.worldPosition, upDirection, smooth)
+
+    /**
+     * ### Rotates the node to face a direction in world-space
+     *
+     * The look direction and up direction cannot be coincident (parallel) or the orientation will
+     * be invalid.
+     *
+     * @param lookDirection The desired look direction in world-space
+     * @param upDirection The up direction will determine the orientation of the node around the direction
+     * @param smooth Whether the rotation should happen smoothly
+     */
+    fun lookAtDirection(
```

Comment:
Maybe it is better to leave the name `lookTowards`? We technically look along a direction not at it :sweat_smile:

Reply:
Done!